src/components/app-configure/access/ChainType.vue

```diff
@@ -0,0 +1,53 @@
+<script lang="ts" setup>
+import { computed, type ComputedRef } from 'vue'
+import { useStore } from 'vuex'
+
+import SettingCard from '@/components/app-configure/SettingCard.vue'
+import VCard from '@/components/lib/VCard/VCard.vue'
+import VRadioGroup from '@/components/lib/VRadioGroup/VRadioGroup.vue'
+import VStack from '@/components/lib/VStack/VStack.vue'
+import { chains } from '@/utils/constants'
```

Comment:
In the constants file, add a chain and option type and use it to type the options.

```
type Chain = "ethereum" | "polygon" | "binance"

type Option<T> = { label: string, value: T }

const CHAIN_OPTIONS: Option<Chain>[] = [
  { label: "Ethereum", value: "ethereum" },
  { label: "Polygon", value: "polygon" },
  { label: "Binance", value: "binance" },
]
```